### ISSUE TITLE
Upgrade Sufia/Curration Concerns dependencies to use Hyrax instead

### ISSUE THREAD

**Issue by murny** [ualbertalib/Hydranorth2#125]
Issue to track against the upgrading of Sufia/CC dependencies to use Hyrax instead (hyrax is a consolidation project of these two dependencies, https://github.com/projecthydra-labs/hyrax).

This is something we need to do sooner then later as this is the way hydra is moving forward into the future with. Hyku (hydra in a box: https://github.com/projecthydra-labs/hyku) has already dropped these dependencies for Hyrax since December.

This will also help eliminate some future work for us that hyrax has already completed. For example with regards to the collection reversal.

Another big reason to get this out of the way as soon as possible is it will unblock this major issue which is preventing works from being created in HN2 currently as well: https://github.com/ualbertalib/Hydranorth2/issues/124
